invenio_github/ext.py

```diff
@@ -84,27 +77,28 @@ def connect_signals():
             from .handlers import account_post_init
 
             account_setup_committed.connect(
-                account_post_init,
-                sender=GitHubAPI.remote._get_current_object()
+                account_post_init, sender=GitHubAPI.remote._get_current_object()
```

Comment:
The fact that we have to access an underscored method somehow tells that we should do it differently?